tx-source/site_core.php

```diff
@@ -752,7 +752,8 @@
 $Definition['Fourth Anniversary'] = 'Fourth Anniversary';
 $Definition['Friends'] = 'Friends';
 $Definition['From'] = 'From';
-$Definition['Full Site'] = 'Full Site';
+$Definition['From'] = 'From';
+$Definition['from the email digest or'] = 'from the email digest or';
```

Comment:
We don't split up translation strings. Never. 

https://success.vanillaforums.com/kb/articles/493-adding-and-editing-locale-translation-strings

https://shopify.engineering/internationalization-i18n-best-practices-front-end-developers

Reply:
@charrondev updated translations and the template to match